packages/@aws-cdk/aws-eks/package.json

```diff
@@ -43,7 +43,8 @@
   ],
   "author": {
     "name": "Amazon Web Services",
-    "url": "https://aws.amazon.com"
+    "url": "https://aws.amazon.com",
+    "organization": true
```

Comment:
Why do we need “organization” to be true? Wouldn’t it make sense to make it the Jsii default?

Reply:
How do I know that some author is an organization versus a person? I would expect organization is a minority, so the default is false...